Tools/AP_Bootloader/board_types.txt

```diff
@@ -112,6 +112,7 @@ AP_HW_MRO_CTRL_ZERO_H7               1023
 AP_HW_MRO_CTRL_ZERO_OEM_H7           1024
 AP_HW_BEASTH7                        1025
 AP_HW_BEASTF7                        1026
+AP_HW_FlywooF745                     1027
```

Comment:
You need to split this into two commits for the two modules. The titles need to reflect the modules, so

Tools: add bootloader and board types

AP_HAL_ChibiOS: add flywoo ....

Reply:
Okay thanks, I made separate commits for the two modules.